incubator/mariadb-galera/values.yaml

```diff
@@ -0,0 +1,50 @@
+## Adfinis SyGroup MariaDB Galera image version
+## ref: https://hub.docker.com/r/adfinissygroup/k8s-mariadb-galera-centos/tags/
+##
+## Default: none
+image: adfinissygroup/k8s-mariadb-galera-centos:v002
```

Comment:
Could you use a meaningful tag? It's better to tell user the version of mariadb in the tag.